### PR TITLE
Updated command to populate user subscriptions to take a list of channels

### PR DESCRIPTION
**Pull request by rhysyngsun**
#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Part of #1974 

#### What's this PR do?
This renames and updates the existing backpopulate command to be more obvious about the purpose and to accept multiple channel names.

#### How should this be manually tested?
Run `./manage.py subscribe_all_users_to_channel NAME [NAME...]` passing channel names

### CODE REVIEW

**Comment by aliceriot** [comment]
I was able to subscribe all my fake users to two channels! nice

File: channels/tasks_test.py [hunk 1]
```diff
@@ -56,19 +56,46 @@ def test_evict_expired_access_tokens():
...
+    )
 
-    assert mock_add_subscriber.call_count == len(users)
+    assert mock_add_subscriber.call_count == (len(usernames) * len(channel_names))
 
-    for user in users:
-        mock_add_subscriber.asset_any_call(user.username, "nochannel")
+    for username in usernames:
+        for channel_name in channel_names:
+            mock_add_subscriber.asset_any_call(username, channel_name)
```
Comment (aliceriot) [inline channels/tasks_test.py]: is this supposed to be `.assert_any_call`?

File: channels/tasks_test.py [hunk 2]
```diff
@@ -56,19 +56,46 @@ def test_evict_expired_access_tokens():
...
+
+    mock_subscribe_user_range_to_channels = mocker.patch(
+        "channels.tasks.subscribe_user_range_to_channels"
+    )
+
+    with pytest.raises(mocked_celery.replace_exception_class):
+        tasks.subscribe_all_users_to_channels.delay(channel_names=channel_names)
+
+    assert mocked_celery.group.call_count == 1
+    list(mocked_celery.group.call_args[0][0])
```
Comment (aliceriot) [inline channels/tasks_test.py]: what's the purpose of this `list` call?

Comment (rhysyngsun) [reply channels/tasks_test.py]: it's a quirk of how the mocked_celery stuff works, see this comment here (which I didn't want to proliferate everywhere): https://github.com/mitodl/open-discussions/blob/5b347f82123c5c4af04e1b02c841b7a9fed8a804/search/tasks_test.py#L254-L256

Comment (aliceriot) [reply channels/tasks_test.py]: ah, I figured it had to do with a generator or something along those lines

**Comment by rhysyngsun** [comment]
@aliceriot fixed the assertion

**Pull request merged**